### PR TITLE
Resolves #2825

### LINKED ISSUE #2825: The part about "Whatif" not working needs to be more prominent.
[Enter feedback here]
The "WhatIf doesn't work for this commandlet" should be at the very top of the page.... 
Also "Whatif" should be an invalid option if it doesn't actually work.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 6dade7d8-f2c8-2ebe-6828-7ff5736f4da0
* Version Independent ID: bd1a3a35-9e2a-25c3-7387-d72fd31f4231
* Content: [New-SmbShare (SmbShare)](https://docs.microsoft.com/en-us/powershell/module/smbshare/new-smbshare?view=windowsserver2022-ps)
* Content Source: [docset/winserver2022-ps/smbshare/New-SmbShare.md](https://github.com/MicrosoftDocs/windows-powershell-docs/blob/master/docset/winserver2022-ps/smbshare/New-SmbShare.md)
* Product: **w10**
* Technology: **windows**
* GitHub Login: @JasonGerend
* Microsoft Alias: **jgerend**

### PR DESCRIPTION
**Pull request by velkovb**
Move WhatIf note in cmdlet description
NOTE: The WhatIf switch does not work with this cmdlet.

### CODE REVIEW

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by get-itips — APPROVED** [review]

File: docset/winserver2012-ps/smbshare/New-SmbShare.md [hunk 1]
```diff
@@ -25,6 +25,8 @@ The **New-SMBShare** cmdlet exposes a file system folder to remote clients as a
 
 To delete a share that was created by this cmdlet, use the Remove-SMBShare cmdlet.
 
+NOTE: The WhatIf switch does not work with this cmdlet.
```
Comment (sdwheeler) [inline docset/winserver2012-ps/smbshare/New-SmbShare.md:28]:
```suggestion
> [!NOTE]
> The WhatIf switch does not work with this cmdlet.
```

File: docset/winserver2012r2-ps/smbshare/New-SmbShare.md [hunk 2]
```diff
@@ -28,6 +28,8 @@ The **New-SMBShare** cmdlet exposes a file system folder to remote clients as a
 
 To delete a share that was created by this cmdlet, use the Remove-SMBShare cmdlet.
 
+NOTE: The WhatIf switch does not work with this cmdlet.
```
Comment (sdwheeler) [inline docset/winserver2012r2-ps/smbshare/New-SmbShare.md:31]:
```suggestion
> [!NOTE]
> The WhatIf switch does not work with this cmdlet.
```

File: docset/winserver2016-ps/smbshare/New-SmbShare.md [hunk 3]
```diff
@@ -29,6 +29,8 @@ The **New-SmbShare** cmdlet exposes a file system folder to remote clients as a
 
 To delete a share that was created by this cmdlet, use the Remove-SmbShare cmdlet.
 
+NOTE: The WhatIf switch does not work with this cmdlet.
```
Comment (sdwheeler) [inline docset/winserver2016-ps/smbshare/New-SmbShare.md:32]:
```suggestion
> [!NOTE]
> The WhatIf switch does not work with this cmdlet.
```

File: docset/winserver2019-ps/smbshare/New-SmbShare.md [hunk 4]
```diff
@@ -29,6 +29,8 @@ The **New-SmbShare** cmdlet exposes a file system folder to remote clients as a
 
 To delete a share that was created by this cmdlet, use the Remove-SmbShare cmdlet.
 
+NOTE: The WhatIf switch does not work with this cmdlet.
```
Comment (sdwheeler) [inline docset/winserver2019-ps/smbshare/New-SmbShare.md:32]:
```suggestion
> [!NOTE]
> The WhatIf switch does not work with this cmdlet.
```

File: docset/winserver2022-ps/smbshare/New-SmbShare.md [hunk 5]
```diff
@@ -29,6 +29,8 @@ The **New-SmbShare** cmdlet exposes a file system folder to remote clients as a
 
 To delete a share that was created by this cmdlet, use the Remove-SmbShare cmdlet.
 
+NOTE: The WhatIf switch does not work with this cmdlet.
```
Comment (sdwheeler) [inline docset/winserver2022-ps/smbshare/New-SmbShare.md:32]:
```suggestion
> [!NOTE]
> The WhatIf switch does not work with this cmdlet.
```

**Review by sdwheeler — CHANGES_REQUESTED** [review]
Please use the Alert box format for notes like this.

**Comment by velkovb** [comment]
> Please use the Alert box format for notes like this.

@sdwheeler I have been advised in the past not to use this syntax as it does not render in PowerShell. Should I start using it?

**Comment by sdwheeler** [comment]
> Should I start using it?

Yes, the Alert boxes are the preferred format.

**Review by sdwheeler — APPROVED** [review]

**Review by dariomws — APPROVED** [review]

**Comment by scanum** [comment]
@tiburd 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**